### PR TITLE
Attempt to avoid flaky test using conditional wait

### PR DESCRIPTION
**Pull request by apupier**
creating PR to have it running on CI which is the only place where the test are failing currently

### CODE REVIEW

**Comment by apupier** [comment]
it ran well 3 times in a row on CI. i will clean the code now

File: src/test/command.stop.test.ts [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+	
+		it("test stop command execution", async function() {
+			executeCommandSpy.resetHistory();
+			await vscode.commands.executeCommand("atlasmap.start");
+			expect(executeCommandSpy.withArgs("atlasmap.start").calledOnce, "AtlasMap start command was not issued").to.be.true;
+
+			await pWaitFor(() => {
+				return createOutputChannelSpy.withArgs("AtlasMap Server").calledOnce;
+			}, 500, 200);
+			await new Promise(resolve => setTimeout(resolve, 15000));
```
Comment (lhein) [inline src/test/command.stop.test.ts]: isn't the waitFor supposed to make the 15sec sleep obsolete?

Comment (apupier) [reply src/test/command.stop.test.ts]: it is the final intent but it requires to find a condition in the wait for that is more accurate. For now, it is only ensuring that some computations have started but no that it has finished.

**Comment by apupier** [comment]
> Seems 1) test ui availability failed.

yes, seems failing each time on CI. Was fine locally. Conditions in wait for are not accurate enough.

**Comment by apupier** [comment]
I keep it on hold for now. I bet more on your current work using promise and callbacks :-)

**Comment by lhein** [comment]
can be closed?

**Comment by apupier** [comment]
yes

**Pull request closed without merging**